src/constraintFunctions.f90

```diff
@@ -239,6 +239,7 @@ subroutine getEnvVarsAtT( t )
     orderedEnvVarNames(8) = 'JFAC'
```

Comment:
Also, does it matter that the index of these varables is not the same number as in the `environmentVariables.config` file?

Reply:
No, it doesn't matter - if I recall correctly, the order here is defined based on which env vars can depend on others (so those that may depend on others are calculated later in the list). It's independent of the order in the input file.

Reply:
All right. I guess the index in `environmentVariables.config` is not really necessary anyway. This can be tidied up as part of #17, as well.